binding/python/examples/theano/cnn.py

```diff
@@ -71,7 +71,7 @@ def momentum(cost, params, learning_rate, momentum):
     updates = []
 
     for p, g in zip(params, grads):
-        mparam_i = theano.shared(np.zeros(p.get_value().shape, dtype=theano.config.floatX))
+        mparam_i = mv_shared(np.zeros(p.get_value().shape, dtype=theano.config.floatX))
```

Comment:
I create such a different function because I think sometimes people want to use multiverso as the backend for just part of the SharedVariables